### PR TITLE
[faq] Deep learning training based on a large number of small files on Hadoop HDFS problem.

### PR DESCRIPTION
**Pull request by YanjieGao**

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -59,3 +59,16 @@ Each queue enforces a limit on the percentage of resources allocated to a user a
 ### Q: How to configure virtual cluster capacity? 
 
 A: Please refer [configure virtual cluster capacity](../pai-management/doc/how-to-write-pai-configuration.md#configure_vc_capacity)
+
+### Q: Deep learning training based on a large number of small files on Hadoop HDFS problem.
```
[the remark this answers is not in the document]
Comment (YanjieGao) [reply docs/faq.md]: added

[the remark this answers is not in the document]
Comment (fanyangCS) [reply docs/faq.md]: could add some examples for PyTorch.

---
In reply to: [214571735](https://github.com/Microsoft/pai/pull/1201#discussion_r214571735) [](ancestors = 214571735)

[the remark this answers is not in the document]
Comment (YanjieGao) [reply docs/faq.md]: survey pytorch hdfs example issue: https://github.com/pytorch/pytorch/issues/5867
The example is very scarce. Haven't seen example for related to this issue.

**Comment by YanjieGao** [comment]
@DengZeshuai this PR is related with your previous issue

**Comment by YanjieGao** [comment]
as there is not example available, write some sample code to illustrate how to achieve  Pytorch customize dataloader read from sequence file of hdfs. also add how to create sequence file by python

**Comment by YanjieGao** [comment]
zeshuai's issue is not related to hdfs small files

**Pull request closed without merging**